apps/web/components/apps/IntegrationListItem.tsx

```diff
@@ -27,6 +28,13 @@ function IntegrationListItem(props: {
   const [highlight, setHighlight] = useState(hl === props.slug);
   const title = props.name || props.title;
 
+  // The highlight is to show a newly installed app, coming from the app's
+  // redirection after installation, so we proceed to show the corresponding
+  // message
```

Comment:
Added this comment to clarify what's going on here per your suggestion @hariombalhara